Dockerfile

```diff
@@ -9,15 +9,15 @@ CMD ["/app/entrypoint.sh", "start"]
 # Git commit SHAs for the build artifact we want to grab.
 # Promote 2018 Week 1 + Fix a copy/paste error on the burnup chart
```

Comment:
Please update the comments. We've used to mention the week and eventual further commits.